kore/src/Kore/Step/Simplification/Rewrites.hs

```diff
@@ -60,16 +59,16 @@ simplifyEvaluatedRewrites ::
     OrPattern RewritingVariableName
 simplifyEvaluatedRewrites first second =
     makeEvaluateRewrites
-        (OrPattern.toPattern first)
-        (OrPattern.toPattern second)
+        first
+        second
 
 makeEvaluateRewrites ::
-    Pattern RewritingVariableName ->
-    Pattern RewritingVariableName ->
+    OrPattern RewritingVariableName ->
+    OrPattern RewritingVariableName ->
     OrPattern RewritingVariableName
 makeEvaluateRewrites first second =
     OrPattern.fromTermLike $
         TermLike.markSimplified $
             mkRewrites
-                (Pattern.toTermLike first)
-                (Pattern.toTermLike second)
+                (OrPattern.toTermLike first)
+                (OrPattern.toTermLike second)
```

Comment:
I created #2623 about this.

Reply:
I'll revert the changes in this file since it's not part of the tests or kore-repl anyway. Is it ok?